packages/cli/commands/validate.js

```diff
@@ -0,0 +1,19 @@
+const {
+  addConfigOptions,
+  addAccountOptions,
+  addOverwriteOptions,
+} = require('../lib/commonOpts');
+const validateTheme = require('./validation/validateTheme');
+
+exports.command = 'validate';
+exports.describe = 'Commands for working with validation';
+
+exports.builder = yargs => {
+  addOverwriteOptions(yargs, true);
```

Comment:
This should be removed since the command doesn't have a need to overwrite like `hs fetch` does.

Reply:
Done in ee4a6dd